drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -127,7 +143,7 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
         default={},
     )
 
-    num_attempts = Integer(
+    attempts = Integer(
```

Comment:
This has never been released previously, correct? Just want to make sure there is no data migration issue.